pkg/test/resourcefixture/testdata/basic/dataplex/v1alpha1/dataplextask/dataplextask-minimal/_http.log

```diff
@@ -406,6 +442,10 @@ OK
   },
   "createTime": "2024-04-01T12:34:56.123456Z",
   "description": "Initial description",
+  "labels": {
```

Comment:
It feels like a bug that we were able to merge controllers with "broken" labels handling.